src/nim_status/lib.nim

```diff
@@ -0,0 +1,20 @@
+import lib/util
+import nimcrypto
+import strutils
+import unicode
+
+const END_OF_MEDIUM = Rune(0x19).toUTF8
+const prefix = END_OF_MEDIUM & "Ethereum Signed Message:\n"
+
+proc hashMessage*(message: string): string =
+  ## hashMessage calculates the hash of a message to be safely signed by the keycard
```

Comment:
Does this get automatically generated as documentation somewhere?

Reply:
I'm not sure, tbh, but I think it's likely!